DXRMapFixups/DeusEx/Classes/DXRFixupM03.uc

```diff
@@ -36,7 +39,20 @@ function PostFirstEntryMapFixes()
             AddActor(class'#var(prefix)CrateUnbreakableSmall', vect(-9463.387695, 3377.530029, 60));
             AddActor(class'#var(prefix)CrateUnbreakableMed', vect(-9461.959961, 3320.718750, 75));
         }
+
+        if (VanillaMaps) {
+            foreach AllActors(class'ElevatorMover', eMover, 'Elevator1') {
+                // When the map first loads, the elevator interpolates to the top, at a normal speed.
+                // This is ordinarilly never seen but can be on a 35/36 start, so set it there instantly.
```

Comment:
I just tried this, skipped to airfield map and went to backtrack to the helibase. The elevator started at the top but then went down, and back up again. Seems a bit buggy.

Reply:
Yeah, I got that bug. That's what the `MoveTime` stuff is there for. It fixed it for me. Strange that it doesn't for you.

Reply:
I think there's something else that triggers the elevator up and down, it might depend how quickly you get to the button